activity_heartbeating/my_activities.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'temporalio/activity'
+
+module ActivityHeartbeating
+  module MyActivities
+    # Activity that demonstrates progress tracking with heartbeating and cancellation
+    class FakeProgress < Temporalio::Activity::Definition
+      def execute(sleep_interval_ms = 1000)
```

Comment:
```suggestion
      def execute(sleep_interval = 1.0)
```

Can also use float seconds to be more Ruby-ish, but ms is fine too